Add INP glossary entry and replace FID references (#36851)
mdn/content: files/en-us/glossary/first_input_delay/index.md

> title: First input delay

Reviewer: I think for these metrics, we should add the initialism to the title so there are results for "FID" in site search

```suggestion
title: First input delay (FID)
```

We have this for CLS already, for example.

Author: Good idea. I added them for some other key perf metrics too.

slug: Glossary/First_input_delay
page-type: glossary-definition
status:
  - deprecated
---

{{GlossarySidebar}}

> **First input delay** (FID) measures the time from when a user first interacts with your site (i.e. when they click a link, tap on a button, or use a custom, JavaScript-powered control) to the time when the browser is actually able to respond to that interaction. It was designed by Google as one of the [Core Web Vital](https://web.dev/articles/vitals) metrics but was replaced by {{Glossary("Interaction to Next Paint")}} (INP) in May 2024.

Reviewer 2: ```suggestion
**First input delay** (FID) measures the time from when a user first interacts with your site (for example, when they click a link, tap on a button, or use a custom, JavaScript-powered control) to the time when the browser is actually able to respond to that interaction. It was designed by Google as one of the [Core Web Vital](https://web.dev/articles/vitals) metrics but was replaced by {{Glossary("Interaction to Next Paint")}} (INP) in May 2024.
```

> **First input delay** (FID) measures the time from when a user first interacts with your site (i.e. when they click a link, tap on a button, or use a custom, JavaScript-powered control) to the time when the browser is actually able to respond to that interaction. It was designed by Google as one of the [Core Web Vital](https://web.dev/articles/vitals) metrics but was replaced by {{Glossary("Interaction to Next Paint")}} (INP) in May 2024.

Reviewer: It may be more effective for readers to use an admonition right after the sidebar macro, like:

```md

> [WARNING]
> **FID** was designed by Google as one of the [Core Web Vital](https://web.dev/articles/vitals) metrics but was replaced by {{Glossary("Interaction to Next Paint")}} (INP) in May 2024.

```
